templates/vue-starter-template/app/components/FrontendDetailPage.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts">
+import { getProductName } from "@shopware/helpers";
+
+const props = defineProps<{
+  navigationId: string;
+}>();
+
+const { search } = useProductSearch();
+const { buildDynamicBreadcrumbs, pushBreadcrumb } = useBreadcrumbs();
+const { apiClient } = useShopwareContext();
+const errorDetails = ref();
+
+const { data, error } = await useAsyncData(
+  `cmsProduct${props.navigationId}`,
+  async () => {
+    const responses = await Promise.allSettled([
+      search(props.navigationId, {
+        withCmsAssociations: true,
+        associations: {
+          seoUrls: {},
+        },
+      }),
+      apiClient.invoke("readBreadcrumb get /breadcrumb/{id}", {
+        pathParams: {
+          id: props.navigationId,
+        },
+      }),
+    ]);
+
+    for (const response of responses) {
+      if (response.status === "rejected") {
+        console.error("[FrontendDetailPage.vue]", response.reason.message);
+        errorDetails.value = response.reason.message;
```

Comment:
we display a single error, maybe concat them?

Reply:
done